docs/howto/writing-migrations.txt

```diff
@@ -183,3 +183,48 @@ the respective field according to your needs.
   Note there is a race condition if you allow objects to be created while this
   migration is running. Objects created after the ``AddField`` and before
   ``RunPython`` will have their original ``uuid``’s overwritten.
+
+.. _controlling_migration_order:
+
+Controlling the order of migrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you auto-generate a migration using the :djadmin:`makemigrations`
+command, Django will take care of making sure the new migration runs
+after the previous ones. It doesn't infer this from the migrations'
+filenames; rather it creates a graph by stating the migrations upon
+which the new auto-generated migration depends. It does this by
+specifying the ``dependencies`` attribute of the ``Migration`` class
+when it auto-generates your migration::
+
+    # -*- coding: utf-8 -*-
+    from django.db import migrations
+
+    class Migration(migrations.Migration):
+
+        dependencies = [
+            ('myapp', '0123_the_previous_migration'),
+        ]
+
+.. versionadded:: 1.8
```

Comment:
Ah, OK, the ticket gave the commit which introduced the property and I concluded via git that this was therefore new in 1.8. I will remove the annotation.

FTR here is my check (which I mistakenly thought to be rigorous :) )

```
$ git branch --all --contains 5826dc52827254f44833acf678f429af94b4d636
  master
* ticket_24882
  remotes/origin/HEAD -> origin/master
  remotes/origin/master
  remotes/origin/stable/1.8.x
  remotes/origin/ticket_24882
  remotes/upstream/master
  remotes/upstream/stable/1.8.x
```